data/base/endpoint/index/all-children.php

```diff
@@ -0,0 +1,63 @@
+<?php
+namespace Elementor\Data\Base\Endpoint\Index;
+
+use Elementor\Data\Base\Endpoint\Index;
+use WP_REST_Request;
+
+/**
+ * class AllChildren, is optional endpoint.
+ * Used in cases where the endpoints are static & there no use of dynamic endpoints( alpha/{id} ), eg:
+ * 'settings' - controller
+ * 'settings/products' - endpoint
+ * 'settings/partners' - endpoint
+ *
+ * When 'settings' is requested, it should return results of all endpoints ( except it self ):
+ * 'settings/products
+ * 'settings/partners'
+ * By running 'get_items' of each endpoint.
+ */
+class AllChildren extends Index {
+	public function get_format() {
+		return $this->controller->get_name() . '/index';
+	}
+
+	/*
+	 * Retrieves a result(s) of all controller endpoint(s), items.
+	 *
+	 * Run overall endpoints of the current controller.
+	 *
+	 * Example, scenario:
+	 * 'settings' - controller
+	 * 'settings/products' - endpoint
+	 * 'settings/partners' - endpoint
+	 * Result:
+	 * [
+	 *  'products' => [
+	 *      0 => ...
+	 *      1 => ...
+	 *  ],
+	 *  'partners' => [
+	 *      0 => ...
+	 *      1 => ...
+	 *  ],
+	 * ]
+	 */
+	public function get_items( $request ) {
+		$response = [];
+
+		foreach ( $this->controller->endpoints as $endpoint ) {
+			// Skip self.
+			if ( $endpoint === $this ) {
+				continue;
+			}
+
+			$result = $endpoint->get_items( new WP_REST_Request() );
```

Comment:
Use Server REST

Reply:
Handled https://github.com/elementor/elementor/pull/12820/commits/6b242981e0b1ec287cc7b9caf3ce14f429f0a4e3

Reply:
@matipojo, maybe its better now to make 'base-route' items methods protected?